dts/arm/nxp/nxp_k6x.dtsi

```diff
@@ -259,16 +259,27 @@
 			status = "disabled";
 		};
 
+#define K_PIN_DEFINE(k_name, k_port, k_pin, k_mux) \
```

Comment:
Would this be a soc specific define or should it reside somewhere else?

Reply:
It should probalby exist in a header like `dts/arm/atmel/pinctrl_atmel_sam.h`